backend/Dockerfile

```diff
@@ -1,11 +1,18 @@
 FROM ubuntu:18.04
 
 RUN apt-get update && apt-get install -y python3 python3-pip libpq-dev gettext
+
 WORKDIR /match4everyone-backend
 COPY requirements.txt /match4everyone-backend/requirements.txt
 RUN pip3 install -r requirements.txt
 COPY requirements.prod.txt /match4everyone-backend/requirements.prod.txt
 RUN pip3 install -r requirements.prod.txt
 
+COPY . .
+RUN django-admin makemessages --no-location --ignore 00_old_m4h_matching_code
```

Comment:
Lets just remove the old code? I think the refactoring is mostly done now.

Reply:
Separate PR or in this one?

Reply:
https://github.com/match4everyone/match4everything/pull/26

Reply:
#26 done, remember to remove this flag @maltezacharias